JoinColony/colonyCDapp
# Fix: position the Stepper buttons above the dotted lines

## Description

| Before | After |
| ---- | ---- |
|  ![Screenshot 2024-06-18 at 16 00 27](https://github.com/JoinColony/colonyCDapp/assets/50642296/4e3a1d70-897a-4079-ab8f-4e4e127b9f93) | ![Screenshot 2024-06-18 at 15 57 31](https://github.com/JoinColony/colonyCDapp/assets/50642296/3aa80442-f74b-49f2-bb56-78d8c66ae9c0) |

## Testing

1. Enable the "Reputation weighted" extension and use the "Testing governance" preset
2. Create a Simple Payment and choose the "Reputation" decision method
3. Support the stake and submit the Max staking amount
4. Run the following commands to forward the time:
```
curl -X POST --data '{"jsonrpc":"2.0","method":"evm_increaseTime","params":[3600],"id":1}' localhost:8545
curl -X POST --data '{"jsonrpc":"2.0","method":"evm_mine","params":[],"id":1}' localhost:8545
```
5. Reload the page
6. Make sure the pill button is above the dotted lines

## Diffs

**Changes** 🏗

* Just a slight update for the Stepper Button's z-index

Resolves #2535#2535
